test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -639,32 +637,39 @@ public interface InstructionUI {
     }
 
 
-    private static final class TimeoutHandler implements ActionListener {
+    private static final class TimeoutHandlerPanel
+            extends JPanel
+            implements ActionListener {
+
+        private static final String PAUSE_BUTTON_LABEL = "Pause";
+        private static final String RESUME_BUTTON_LABEL = "Resume";
 
-        private static final String PAUSED_LABEL_SUFFIX = " PAUSED";
-        private static final String PAUSE_BUTTON_LABEL = "Pause Timer";
-        private static final String RESUME_BUTTON_LABEL = "Resume Timer";
         private long endTime;
-        private long pauseTimeLeft = 0L;
+        private long pauseTimeLeft;
 
         private final Timer timer;
 
         private final JLabel label;
         private final JButton button;
 
-        public TimeoutHandler(final JLabel label,
-                              final JButton button,
-                              final long testTimeOut) {
-            endTime = System.currentTimeMillis() + TimeUnit.MINUTES.toMillis(testTimeOut);
+        public TimeoutHandlerPanel(final long testTimeOut) {
+            endTime = System.currentTimeMillis()
+                    + TimeUnit.MINUTES.toMillis(testTimeOut);
+
+            label =  new JLabel("", JLabel.CENTER);
+            button = new JButton(TimeoutHandlerPanel.PAUSE_BUTTON_LABEL);
+
+            button.setFocusPainted(false);
```

Comment:
I wonder if it's the default focus owner? Should we request focus to `Pass` instead?

Reply:
> I wonder if it's the default focus owner? 

It is(e.g. with `setFocusPainted(true)`):
![image](https://github.com/openjdk/jdk/assets/77687766/b47e9011-c47d-437d-b9b0-0e4ac88a5c6f)

> Should we request focus to Pass instead?

Probably not, to avoid accidentally passing the test, while hitting the pause is harmless.

Reply:
Not so sure… Pause is not the main control. I guess the focus was in the instructions previously, we could preserve it. It may be not worth it though.

Reply:
It's not clear why we need a focus on the field with instructions.
The focus on the pause button makes sense:
Suppose someone is running many, many tests and decides to take a tea break, he just presses the space bar without moving the mouse.
(if he knows  that the focus is on the pause button, `setFocusPainted(false)` :-) )

Anyway, if this bothers anyone we can come back to it later.